pkg/util/utils.go

```diff
@@ -111,10 +111,20 @@ func (m MultiError) ToError() error {
 // EscapeStringForSSHUse will parse string and replace
 // shell special characters with escape
 func EscapeStringForSSHUse(s string) string {
-	r := strings.NewReplacer("$", "\\$",
+	r := strings.NewReplacer(
+		"$", "\\$",
 		`"`, `\"`,
 		"`", "\\`",
-		`\`, `\\`)
+		`\`, `\\`,
+		`(`, `\(`,
+		`)`, `\)`,
+		`{`, `\{`,
```

Comment:
>  The solution is to put the password into single quotes (instead of double) to prevent substitution within the shell.

I did that and I am getting same error message as mentioned in the issue. Doing the explicit escape as done in this PR fix that. Same for `;` too.